pygmt_paper_numpy_datetime.ipynb

```diff
@@ -0,0 +1,78 @@
+      "text/plain": [
+       "<IPython.core.display.Image object>"
+      ]
+     },
+     "metadata": {
+      "image/png": {
+       "width": 500
+      }
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import datetime\n",
+    "\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import pygmt\n",
+    "\n",
+    "fig = pygmt.Figure()\n",
+    "\n",
+    "fig.basemap(\n",
+    "    projection=\"X12c/6c\",\n",
+    "    region=[datetime.date(2016, 1, 1), datetime.date(2026, 12, 31), 0, 1000],\n",
```

Comment:
Maybe set ymin to -50 so that we don't need to set `no_clip=True`.
```suggestion
    "    region=[datetime.date(2016, 1, 1), datetime.date(2026, 12, 31), -50, 1000],\n",
```